ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -596,6 +596,364 @@ export default {
+      // 8FC: Cold Blade: -2
+      // Positional data in statusEffectsParams is often not up to date, use promise
+      //
+      // Trigger delayed until after Blade Of Entropy happens about ~100ms after
+      // to get left/right cleave info
+      // Ignoring Trinity Avowed due to Environment 'randomly' refreshing its buff
+      netRegex: NetRegexes.gainsEffect({ target: 'Avowed Avatar', effectId: ['8F9', '8FA', '8FB', '8FC'], capture: false }),
+      netRegexDe: NetRegexes.gainsEffect({ target: 'Spaltteil der Eingeschworenen', effectId: ['8F9', '8FA', '8FB', '8FC'], capture: false }),
+      netRegexFr: NetRegexes.gainsEffect({ target: 'Clone De La Trinité Féale', effectId: ['8F9', '8FA', '8FB', '8FC'], capture: false }),
+      netRegexJa: NetRegexes.gainsEffect({ target: 'アヴァウドの分体', effectId: ['8F9', '8FA', '8FB', '8FC'], capture: false }),
+      delaySeconds: 0.5,
+      durationSeconds: 9.5,
+      suppressSeconds: 1,
+      promise: async (data, _, output) => {
+        const trinityLocaleNames = {
+          en: 'Trinity Avowed',
+          de: 'Trinität Der Eingeschworenen',
+          fr: 'Trinité Féale',
+          ja: 'トリニティ・アヴァウ',
+        };
+
+        const avatarLocaleNames = {
+          en: 'Avowed Avatar',
+          de: 'Spaltteil der Eingeschworenen',
+          fr: 'Clone De La Trinité Féale',
+          ja: 'アヴァウドの分体',
+        };
+
+        // select the Trinity and Avatars
+        let combatantNameBoss = null;
+        let combatantNameAvatar = null;
+        combatantNameBoss = trinityLocaleNames[data.parserLang];
+        combatantNameAvatar = avatarLocaleNames[data.parserLang];
+
+        let combatantDataBoss = null;
+        let combatantDataAvatars = null;
+        if (combatantNameBoss) {
+          combatantDataBoss = await window.callOverlayHandler({
+            call: 'getCombatants',
+            names: [combatantNameBoss],
+          });
+        }
+        if (combatantNameAvatar) {
+          combatantDataAvatars = await window.callOverlayHandler({
+            call: 'getCombatants',
+            names: [combatantNameAvatar],
+          });
+        }
+
+        // if we could not retrieve combatant data, the
+        // trigger will not work, so just resume promise here
+        if (combatantDataBoss === null) {
+          console.error(`Trinity Avowed: null data`);
+          data.safeZone = null;
+          return;
+        }
+        if (!combatantDataBoss.combatants) {
+          console.error(`Trinity Avowed: null combatants`);
+          data.safeZone = null;
+          return;
+        }
+        if (combatantDataAvatars === null) {
+          console.error(`Avowed Avatar: null data`);
+          data.safeZone = null;
+          return;
+        }
+        if (!combatantDataAvatars.combatants) {
+          console.error(`Avowed Avatar: null combatants`);
+          data.safeZone = null;
+          return;
+        }
+        if (combatantDataAvatars.combatants.legnth < 3) {
```

Comment:
```suggestion
        if (combatantDataAvatars.combatants.length < 3) {
```

Do you get more here?

Reply:
Currently, it should get more than 3, but if we change the trigger used to do the promise, we can get it so that only three (or 4 if boss included) are checked.